templates/jlink/jlink-builder/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenShift-JLink PoC
+
+Try it out:
+
+## Stage 1: create ubi9 jlink imagestream and push a ubi9 image with jmods installed to it.
+
+You need:
+
+1. Access to an OpenShift instance, such as crc
+
+Steps to produce the imagestream and image:
+
+1. Switch to the openshift project and install the template
+
+oc project openshift
+oc create -f templates/jlink/jlink-builder/jlink-builder-template.yaml
```

Comment:
Indent all the example input/output lines by 4 or more spaces and they'll be rendered in monotype